lighthouse-core/report/html/renderer/report-ui-features.js

```diff
@@ -260,26 +254,6 @@ class ReportUIFeatures {
     this._copyAttempt = false;
   }
 
-  _setupMetricDescriptionToggleElements() {
-    const metricDescriptionToggleEl = this._document.querySelector('.lh-metrics-toggle__input');
-    // No metrics if performance category wasn't run.
-    if (!metricDescriptionToggleEl) return;
-
-    this.metricDescriptionToggleEl = /** @type {HTMLInputElement} */ (metricDescriptionToggleEl);
-    this.metricAuditGroup = this._dom.find('.lh-audit-group--metrics', this._document);
-    this.metricDescriptionToggleEl.addEventListener('input', this._toggleMetricDescription);
-    this.metricAuditGroup.addEventListener('click', e => {
-      const el = /** @type {HTMLElement} */ (e.target);
-      if (el.closest('.lh-metric__title')) this.metricDescriptionToggleEl.click();
```

Comment:
we lose this click behavior now

IMO, it was a little wonky anyhoo so not the biggest loss but something to consider 🤷‍♂

Reply:
we agreed to remove this thing separately. connor's PR also nukes it
https://github.com/GoogleChrome/lighthouse/pull/8785/files#diff-10f7ab3e3a96491edf29819a09f5b45fL271